components/CaseStudiesProject.tsx

```diff
@@ -1,37 +1,59 @@
+"use client";
+
+import { motion } from "framer-motion";
 import { Card } from "@nextui-org/card";
-import { Link } from "@nextui-org/link";
 import Image from "next/image";
+import Link from "next/link";
+import Tilt from "react-parallax-tilt";
 
 const CaseStudiesProject = ({
   project,
 }: {
-  project: { name: string; description: string; image: string; color: string };
+  project: { name: string; description: string; image: string };
 }) => {
+  const colorVariants: { [key: string]: string } = {
+    morrent: "bg-morrentLight",
+    jobit: "bg-jobit",
+    hipnode: "bg-hipnode",
+    filmpire: "bg-filmpire",
+  };
```

Comment:
Please move this to a separate component and have it imported. This will make it cleaner and improve readability.

Reply:
While working on another branch, I've just found that without the colorVariants in the component, the color doesn't get applied because TailwindCSS doesn't allow dynamic class names. Therefore, I will restore that in another branch.